font-codegen/src/fields.rs

```diff
@@ -1115,12 +1115,16 @@ impl Field {
     fn from_obj_requires_offset_data(&self, in_record: bool) -> bool {
         match &self.typ {
             _ if self.attrs.to_owned.is_some() => false,
+            FieldType::Offset {
+                target: Some(OffsetTarget::Array(_)),
+                ..
+            } => true,
             FieldType::Offset { .. } => in_record,
             FieldType::ComputedArray(_) | FieldType::VarLenArray(_) => true,
             FieldType::Struct { .. } => true,
             FieldType::Array { inner_typ } => match inner_typ.as_ref() {
```

Comment:
Do you have to have a nested match here or could you have multiple parent level matches?

Reply:
Unfortunately we need to nest because the inner type is in a `Box`, and so we can't match it directly without the `as_ref` call. :(